src/utils/private.ts

```diff
@@ -184,6 +185,7 @@ export const getWideConfig = async (context: Context) => {
     defaultLabels: getDefaultLabels(configs),
     promotionComment: getPromotionComment(configs),
     registerWalletWithVerification: getRegisterWalletWithVerification(configs),
+    staleBountyTime: parsedDefault["stale-bounty-time"],
```

Comment:
Pls check how the bot's config params are extracted [here](https://github.com/ubiquity/ubiquibot/blob/7fd7d3b83aaab0b19673d9c601937d9fdeb3f18c/src/utils/helpers.ts). There is a hierarchy of configs where the latter rewrites the former one:
1. Ubiquibot [default config](https://github.com/ubiquity/ubiquibot/blob/development/ubiquibot-config-default.json)
2. Partner organization wide config ([example](https://github.com/rndquu-org/ubiquibot-config/blob/main/.github/ubiquibot-config.yml))
3. Repository wide config ([example](https://github.com/ubiquity/ubiquibot/blob/development/.github/ubiquibot-config.yml))

Pls refactor the `stale-bounty-time` config extraction [accordingly](https://github.com/ubiquity/ubiquibot/blob/7fd7d3b83aaab0b19673d9c601937d9fdeb3f18c/src/utils/helpers.ts)